FromListToTuple.py

```diff
@@ -0,0 +1,4 @@
+my_list = [34, 82.6, "Darth Vader", 17, "Hannibal"]
+
+my_tuple=(my_list[0], my_list[len(my_list)-1], len(my_list))
```

Comment:
You can access the last element using **my_list[-1]** instead of **my_list[len(my_list)-1]**

Reply:
What happend is my_list = [] ?

Reply:
Done